src/main/java/com/vadmack/mongodbtest/util/SortService.java

```diff
@@ -0,0 +1,32 @@
+package com.vadmack.mongodbtest.util;
+
+import com.vadmack.mongodbtest.exception.ValidationException;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+@Service
+public class SortService {
+
+    public Sort createSort(String[] sortBy) {
+        return Sort.by(
+                Arrays.stream(sortBy)
+                        .map(sort -> sort.split(":", 2))
```

Comment:
What happens if client provides incorrect "sort" parameter (without ":", or direction is not expected character, or sort field diesn't exists)? 

I suggest to validate and parse the value using regular expression